scripts/ccf/sandbox_local/scale-nodes.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+ccf-sandbox-local-nodes() {
+    curl -s $KMS_URL/node/network/nodes \
+        --cacert $KMS_SERVICE_CERT_PATH | jq '.nodes'
+}
+
+ccf-sandbox-local-node() {
+    curl -s $KMS_URL/node/network/nodes/$1 \
+        --cacert $KMS_SERVICE_CERT_PATH | jq
+}
+
+status-is() {
+    jq --arg status "$1" '[.[] | select(.status == $status)]'
```

Comment:
what is this argument? Maybe good to add a comment

Reply:
Sorry what do you mean? You're wondering what the positional argument to `status-is` is? If so, isn't it obvious since the function is called `status-is` and it's immediately compared to the `.status` field to select elements?